app/src/main/java/cl/freedom/wheatermvvm/data/CovidDatabase.kt

```diff
@@ -0,0 +1,39 @@
+package cl.freedom.wheatermvvm.data
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import cl.freedom.wheatermvvm.dashboard.data.CurrentCovidDao
+import cl.freedom.wheatermvvm.dashboard.data.CurrentCovidEntry
+
+@Database(
+    entities = [CurrentCovidEntry::class],
+    version = 1,
+    exportSchema = true
+)
+abstract class CovidDatabase : RoomDatabase() {
+    abstract fun currentCovidDato() : CurrentCovidDao
```

Comment:
What does this variable means or do ? "currentCovidDato". Please refactor the name, it's confusing